values.yaml

```diff
@@ -1,8 +1,16 @@
 version: 1.15.0
 
+
+#Note: If LAGO_DOMAIN is set, these variables (apiUrl & frontUrl) will be overridden.
 # apiUrl: mydomain.dev
 # frontUrl: mydomain.dev
 
+# By setting the LAGO_DOMAIN variable, your application will be available directly on the specified domain. 
+# For example, if LAGO_DOMAIN is set to getmydomain.dev, your GetLago instance will be accessible at https://getmydomain.dev,
+# and your API at https://getmydomain.dev/api/api/v1. Note that adding this variable changes the domain structure:
```

Comment:
I just think about it, but would be awesome if we can manage to not use `/api/api`, do you think its possible?